README.md

```diff
@@ -21,14 +21,9 @@
 
 **nf-core/pairgenomealign** is a bioinformatics pipeline that aligns one or more _query_ genomes to a _target_ genome, and plots pairwise representations.
 
-<img src= "assets/tube_map.svg">
+<img src= "docs/images/pairgenomealgn-logo.png">
```

Comment:
```suggestion
![pairgenomealign-tubemap.png](docs/images/pairgenomealgn-logo.png)
```
needs to be a markdown image link to be rendered correctly on the website.
Also the tubemap should not replace the logo (which we use for example in the mulitqc report, please restore the old logo there and add this graph as `pairgenomealign-tube-map.png`
(there is also a typo in the current file name)

Reply:
I did not see your suggestion and just pushed an equivalent change.